locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -2623,7 +2645,7 @@ msgstr "Commencer « <em>%(book_title)s</em> »"
 #: bookwyrm/templates/snippets/shelve_button/want_to_read_modal.html:5
 #, python-format
 msgid "Want to Read \"<em>%(book_title)s</em>\""
-msgstr "A envie de lire « <em>%(book_title)s</em> »"
+msgstr "Ajouter « <em>%(book_title)s</em> » aux envies de lecture"
```

Comment:
I don’t feel like this change is making the phrasing easier to read. :|

« J’ai envie de lire… » ?

Reply:
As a modal title, I think it's better to make it as a clear action and not a desire

<img width="708" alt="image" src="https://user-images.githubusercontent.com/1781070/121027066-f68f0a00-c7a6-11eb-8a7d-7b6507091499.png">

Reply:
Ha yes; perfect.